app/models/restaurant.rb

```diff
@@ -0,0 +1,22 @@
+# == Schema Information
+#
+# Table name: restaurants
+#
+#  id                :integer          not null, primary key
+#  name              :string           not null
+#  cuisine           :string           not null
+#  rating            :integer          default(0)
+#  accepts_10bis     :boolean          default(FALSE)
+#  address           :string           not null
+#  max_delivery_time :integer          default(120)
+#  created_at        :datetime         not null
+#  updated_at        :datetime         not null
+#  reviews_count     :integer          default(0)
+#  reviews_sum       :integer          default(0)
+#
+
+class Restaurant < ApplicationRecord
+  has_many :reviews, dependent: :destroy
+
+  validates_presence_of :name, :cuisine, :address
+end
```

Comment:
You can add a bit more validations here, such as validating that accepts_ten_bis is boolean, also validation on rating, reviews_count & reviews_sum (that they are larger than 0 for example)

Reply:
👍